rules/test_host_app/BUILD.bazel

```diff
@@ -7,17 +7,17 @@ exports_files([
 
 [
     ios_application(
-        name = "iOS-%s-AppHost" % version,
```

Comment:
want to learn why this change is needed, no more support of this `%` syntax?

Reply:
No it's still supported, but using the explicit `format` method reads clearer

Reply:
i see i thought this is a better syntax, will use `format` in future instead of this